explainer/Explainer11.md

```diff
@@ -0,0 +1,287 @@
+# WoT Thing Description (TD) 1.1 Explainer (DRAFT - WIP)
+This is an updated version of the [Explainer for TD 1.0](Explainer.md).
```

Comment:
Since this is an explainer for the TD spec specifically, I guess I won't need to add extra material for Discovery, etc.  However, we should have an overall introduction somewhere, perhaps for Architecture, that can then refer to the individual explainers for details.  When/if we do that, though, a link back to the "general introduction" (e.g. the Architecture explainer) might be helpful, and some of the more general comments in this doc (not sure what those might be yet) could be handled there.  Although at this point, I think getting the TD explainer out the door and the wide review process going is the priority, so...